cmd/fleet/main.go

```diff
@@ -72,7 +72,7 @@ func applyDevFlags(cfg *config.FleetConfig) {
 		cfg.Prometheus.BasicAuth.Password = "insecure"
 	}
 
-	cfg.S3 = config.S3Config{
+	cfg.CarvesS3 = config.S3Config{
```

Comment:
should we also add a default dev config for software installers?

Reply:
Ooh good point! I always forget that that's managed through code.